src/main/java/com/cruru/applicant/service/ApplicantService.java

```diff
@@ -45,4 +52,24 @@ private ApplicantResponse toApplicantResponse(Applicant applicant) {
                 applicant.getCreatedDate()
         );
     }
+
+    public ApplicantDetailResponse findDetailById(long id) {
+        Applicant applicant = applicantRepository.findById(id)
+                .orElseThrow(ApplicantNotFoundException::new);
+        List<Answer> answers = answerRepository.findAllByApplicantId(id);
```

Comment:
```suggestion
        Applicant applicant = applicantRepository.findById(id)
                .orElseThrow(ApplicantNotFoundException::new);
        List<Answer> answers = answerRepository.findAllByApplicant(applicant);
```
개인적으로 찾아온 객체가 있다면, 그 객체를 활용해서 찾는 것이 코드 흐름상 더 나아보인다고 생각하는데요. 어떻게 생각하시나요?

Reply:
변경했습니다~